lib/style.css

```diff
@@ -56,6 +56,20 @@ body {
   background-color: <<BACK_PRIMARY>>;
 }
 
+#player.material paper-icon-button.playing::after {
+  content: '';
+  display: block;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  margin-top: -1.2em;
+  height: 2.4em;
+  width: 2.4em;
```

Comment:
I can change to `::before` and can then lose the `z-index` but the margins are still required to centre the fake background.  If you imagine how this works I am putting a white square inside a circle.  The square therefore has to be smaller than the circle so it has to be centred inside the circle :)